src/components/NavBar/Navbar.tsx

```diff
@@ -0,0 +1,28 @@
+import Web3Status from 'components/Web3Status'
+import { useWindowSize } from 'hooks/useWindowSize'
+
+import { breakpoints } from '../../nft/css/sprinkles.css'
+import * as styles from './Navbar.css'
+
+const MobileNavbar = () => {
+  return (
+    <>
+      <nav className={styles.nav} />
+      <div className={styles.mobileWalletContainer}>
+        <Web3Status />
+      </div>
+    </>
+  )
+}
+
+const Navbar = () => {
+  const { width: windowWidth } = useWindowSize()
+
+  if (windowWidth && windowWidth < breakpoints.desktopXl) {
```

Comment:
this check should happen inside the styled component, with visibility hidden under a certain breakpoint

Reply:
what would be the benefits of doing this over current implementation?

Reply:
In general, we should be handling breakpoints in CSS rather than separating them into components. say we want to have 5 different breakpoints, it would be messy to have 5 different components to support that.

Reply:
also technically you could hit this breakpoint on web by making the screen small enough, so it's not mobile specific

Reply:
i think it's already doing this right? https://github.com/Uniswap/interface/pull/4365/files#diff-366a560da382b959d44ac866e3febea7ad155874ff9bd316ad4820edceec8ef1R25

Reply:
We usually handle breakpoints in css however in this case there is a major design difference between the navbar seen on mobile vs desktop where several tabs are removed, the chain switcher swaps sides, the wallet drops to the bottom, and the searchbar becomes a collapsible icon. This is best seen on the full phase 0 navbar PR. This is why we did this major split between the two components.